app/models/organization.rb

```diff
@@ -5,5 +5,8 @@ class Organization < ActiveRecord::Base
   translates :description
   mount_uploader :logo, LogoUploader
 
+  has_many   :sponsorships
+  has_many   :sponsors, through: :sponsorships, source: :event
```

Comment:
Or `has_many   :sponsored_events, through: :sponsorships, source: :event`
to be more precise